examples/open_loop_control_fetch/run.py

```diff
@@ -13,92 +13,61 @@
+    parser.add_argument("--actions_file", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "actions.pkl")))
+    parser.add_argument("--images_dir", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "images")))
+    parser.add_argument("--scene_id", default="apartment_0000")
     parser.add_argument("--save_images", action="store_true")
     parser.add_argument("--benchmark", action="store_true")
     args = parser.parse_args()
 
     np.set_printoptions(linewidth=200)
 
+    # read pre-recorded actions from the actions file
+    df = pd.read_pickle(args.actions_file)
+
     # load config
     config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
 
-    # get pregenerated actions for fetch urdf agent
-    df = pd.read_csv(args.actions_file)
-
-    # do some config modifications based on the rendering mode
-    if args.rendering_mode == "baked":
-        rendering_mode_map_str = "_bake"
-        config.defrost()
-        config.SIMULATION_CONTROLLER.CAMERA_SENSOR.FINAL_COLOR_INDIRECT_LIGHTING_INTENSITY = 1.0
-        config.freeze()
-    elif args.rendering_mode == "raytracing":
-        rendering_mode_map_str = "_rtx"
-        config.defrost()
-        config.SIMULATION_CONTROLLER.CAMERA_SENSOR.FINAL_COLOR_INDIRECT_LIGHTING_INTENSITY = 0.0
-        config.freeze()
-    else:
-        assert False
-
-    # change config based on current scene
+    # update scene_id
     config.defrost()
-
-    if args.scene_id == "starter_content_0000":
-        # starter_content_0000 doesn't need a rendering mode when referring to its map
-        config.SIMULATION_CONTROLLER.SCENE_ID = args.scene_id
-        config.SIMULATION_CONTROLLER.MAP_ID   = args.scene_id
-
-        # starter_content_0000 has scene-specific config values
-        scene_config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "scene_config.starter_content_0000.yaml")
-
-    elif args.scene_id == "kujiale_0000":
-        config.SIMULATION_CONTROLLER.SCENE_ID = args.scene_id
-        config.SIMULATION_CONTROLLER.MAP_ID   = args.scene_id + rendering_mode_map_str
-
-        # kujiale_0000 has scene-specific config values
-        scene_config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "scene_config.kujiale_0000.yaml")
-
-    config.merge_from_file(scene_config_file)
+    config.SIMULATION_CONTROLLER.SCENE_ID = args.scene_id
     config.freeze()
-
+    
     # create Env object
     env = spear.Env(config)
 
-    # reset the simulation to get the first observation
+    # reset the simulation
     obs = env.reset()
 
     if args.benchmark:
         start_time_seconds = time.time()
     else:
         if args.save_images:
-            for render_pass in config.SIMULATION_CONTROLLER.CAMERA_AGENT.CAMERA.RENDER_PASSES:
-                render_pass_dir = os.path.realpath(os.path.join(args.image_dir, render_pass))
+            index = 0
```

Comment:
Move `index = 0` to be immediately above the for loop where it is used, so I can understand what is going on without needing to look at as much code.

Reply:
Updated the code.